srcpkgs/docker-engine/template

```diff
@@ -0,0 +1,60 @@
+# Template file for 'docker-engine'
+pkgname=docker-engine
+version=20.10.5
+revision=1
+archs="aarch64* armv[67]l* x86_64* ppc64le*"
+create_wrksrc=yes
+build_style=go
+go_import_path="github.com/docker/docker"
+hostmakedepends="pkg-config tar"
+makedepends="libbtrfs-devel device-mapper-devel libseccomp-devel"
+depends="containerd runc iptables xz git"
+short_desc="Pack, ship and run any application as a lightweight container - engine"
+maintainer="PWA COLLECTIVE <mail@pwa.cya.cx>"
+license="Apache-2.0"
+homepage="http://www.docker.io"
+system_groups="docker"
+
+_moby_commit=363e9a88a11be517d9e8c65c998ff56f774eb4dc	# https://github.com/moby/moby/commits/v$version
```

Comment:
Just `export DOCKER_GITCOMMIT=363e9[...]` in `do_build` and drop this pointless variable.

Reply:
I'm going to leave these in as per your other comment about moving the commit variables up under `revision`.